### DISCUSSION TITLE
Unixsocket support

### DISCUSSION THREAD

**Discussion by GrgoMariani** [jchristn/WatsonWebsocket#122] [Q&A]
Is there any chance we might see support for Unix sockets in the future?

**Comment by jchristn** [comment]
Hi @GrgoMariani are you referring to UNIX Domain Socket?  If so, unfortunately no.  This library relies on the underlying ```HttpListener``` implementation.  It does not provide its own, so we can only leverage what that implementation leverages.

**Reply by GrgoMariani** [reply]
Yes indeed. Unfortunately the UNIX Domain Socket has no traction with the C# WebSocket implementations I guess. Too bad as I find it rather useful in combination with nginx.
